apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -289,7 +273,9 @@ NetSimLocalClientNode.prototype.disconnectRemote = function (onComplete) {
 
     self.myWire = null;
     // Trigger an immediate router update so its connection count is correct.
-    self.myRouter.update(onComplete);
+    self.myRouter.update(function (err, result) {
+      onComplete(result);
+    });
```

Comment:
Up to you. I think it would be good to get to a place where they are all consistent, but if you think it makes more sense to make that transition over time, that's fine by me.